tests/unit/dataset/loader/test_hf_conversation_loader.py

```
@@ -242,6 +251,123 @@ async def test_attaches_first_image_from_list(self, user_config):
+            user_config=user_config,
+            hf_dataset_name="lmarena-ai/VisionArena-Chat",
+            hf_split="train",
+            conversation_column="conversation",
+            message_content_key="content",
+            image_column="images",
+        )
+        data = {
+            "dataset": [
+                {
+                    "conversation": [{"role": "user", "content": "Path only"}],
+                    "images": [{"bytes": None, "path": "/tmp/some_image.jpg"}],
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/some_image.jpg"

(S108)